src/styles/styles.js

```diff
@@ -3311,6 +3311,15 @@ const styles = {
         fontWeight: fontWeightBold,
         lineHeight: undefined,
     },
+    expensifyQrLogo: {
```

Comment:
Let's also add blank lines above these, for consistency with the rest of the file.

```suggestion

    expensifyQrLogo: {
```